File: Assignmentno17/task_2.java
import java.util.*;

class Arithematic
{
    void Palindrom(int iNo)
    {
        
        int temp = iNo;
        int rev =0;
        while(iNo!=0)
        {
            rev = rev*10+(iNo%10);
            iNo = iNo/10;
        }
        if(temp == rev)
        {
            System.out.println("the number is palindrom");
        }
        else
        {
            System.out.println("the number is not palindrom");

        }

    }
}

class task_2
{
    public static void main(String A[])
    {
        int iValue =0;
        Scanner sobj = new Scanner(System.in);
        Arithematic aobj = new Arithematic();

        System.out.println("Enter the number:");
        iValue=sobj.nextInt();
        aobj.Palindrom(iValue);


    }
}

File: Assignmentno70/task_2.c
#include<stdio.h>
int Addition(int iValue)
{
     static int iSum = 0;
    if(iValue == 0)
    {
        return iSum;
    }

    iSum = iSum + (iValue%10);

    return Addition(iValue/10);


}

int main()
{
    int iNo = 0;
    int iRet = 0;
    printf("Enter the number:\n");
    scanf("%d",&iNo);
    iRet = Addition(iNo);
    printf("The addition is:%d\n",iRet);

    return 0;
}